third_party/proxy/meta_proxier.go

```diff
@@ -40,7 +40,7 @@ package proxy
 import (
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/klog"
```

Comment:
this is not changed to klog v2?

Reply:
Maybe I accidentally revert changes for this file. I will fix this. Thanks